Makefile

```diff
@@ -4,7 +4,7 @@ format:
 	mvnd spring-javaformat:apply
 
 build:
-	mvnd spring-javaformat:apply verify
+	mvnd spring-javaformat:apply verify spring-javaformat:apply
```

Comment:
why we need this again in the same command line?

Reply:
First we format, then in verify phase code generation runs which can generate unformatted code, so that we need to format again. 

Alternative would be to generate code in `generated-sources` but then, if generated file changes for whatever reason (like SDK update) its easy to miss it.

Reply:
Figured out a better way - class is now generated into a separate module - `spring-cloud-aws-s3-cross-region` which is not a subject of spring-formatting or checkstyle (as its not easy to generate class that matches these conventions).